src/assets/testimonials-data.js

```diff
@@ -0,0 +1,40 @@
+const testimonialsData = [

```

Comment:
Loving this 🥳  Wil je dit in een mapje zetten genaamd `constants` op dezelfde hoogte als `assets`?